modules/library.js

```diff
@@ -0,0 +1,38 @@
+import Books from './book.js';
+
+const bookList = document.querySelector('.book-list');
+const titleInput = document.querySelector('#input_title');
+const authorInput = document.querySelector('#input_author');
+
+export default class Library {
+  constructor() {
+    this.Library = JSON.parse(localStorage.getItem('collections')) || [];
+  }
+
+  addBook() {
+    const newBook = new Books();
+    newBook.title = titleInput.value;
+    newBook.author = authorInput.value;
+    this.Library.push(newBook);
+    localStorage.setItem('collections', JSON.stringify(this.Library));
+    this.showBook();
+  }
+
+  showBook() {
+    bookList.innerHTML = '';
+    this.Library.forEach((collection, i) => {
+      bookList.innerHTML += `<div id="${i}" class="books">
+            <p class="book-info">
+                <span class="title">"${collection.title}" by ${collection.author}</span>
+            </p>
+            <button class="remove-btn">Remove</button>
+    </div>`;
+    });
+  }
+
+  removeBook(book) {
+    book.parentElement.remove();
+    this.Library = this.Library.filter((collection, i) => i !== Number(book.parentElement.id));
+    localStorage.setItem('collections', JSON.stringify(this.Library));
+  }
+}
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/212851360-516d45ce-7f2b-4b97-b798-4bf7db4b2d2d.png)

- [ ] Your methods are not really ES6 methods. You can see in the image that I have changed your methods to arrow methods. Kindly change your methods as well to follow the ES6 :blush:
- [ ] Your Application is a Single Page Application but when i add or remove a book then your page reloads which is not expected behavior from an Single Page Application :thinking: Kindly make your application behave like Single Page Application :+1:

Reply:
Hi! I hope you are doing well. I have done first requirement in library.js file. But I need your suggestion for second requirement. If I remove window.function.reload() function then remove function doesn't behave right. When I remove two books together and refresh the page then it removes second object wrong.

Reply:
Hey @MAleemH
You need already have a render method in your class you just need to use that render method after adding or removing a book :+1: I hope it will help you otherwise you can reach out to me on slack so we can fix this in one call :+1: